typescript/packages/well-completions-plot/src/components/WellsPlot.tsx

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable react-hooks/exhaustive-deps */ // remove when ready to fix these.
+
+import React from "react";
+import {
+    AttributeType,
+    PlotData,
+    SortBy,
+    SortByString,
+    WellPlotData,
+} from "../types/dataTypes";
+import { capitalizeFirstLetter } from "../utils/stringUtil";
+import { useTooltip } from "./TooltipProvider";
+import { Padding, PlotLayout } from "../types/layoutTypes";
+
+interface WellTooltipContentProps {
+    name: string;
+    earliestCompDate: string;
+    attributes: Record<string, AttributeType>;
+}
+
+const WellTooltipContent: React.FC<WellTooltipContentProps> = (
+    props: WellTooltipContentProps
+) => {
+    return (
+        <table style={{ color: "#fff" }}>
+            <tbody>
+                {/* earliest completion date */}
+                <tr key={`well-tooltip-${props.name}-earliest-comp`}>
+                    <td>
+                        <b>
+                            {capitalizeFirstLetter(
```

Comment:
Still necessary with the enum -> string mapping?

Reply:
The `SortByEnumToStringMapping` is used other places where lower case is preferred, so I'll keep lower case in mapping and use `capitalizeFirstLetter` here.